app/src/main/java/co/condorlabs/customcomponents/customedittext/BaseEditTextFormField.kt

```diff
@@ -267,8 +266,4 @@ open class BaseEditTextFormField(context: Context, private val attrs: AttributeS
             clearError()
         }
     }
-
-    fun getInputType() = inputType
-
-    fun getRegex() = regex
```

Comment:
regex is a set, we should not create a String to capture the element provided via XM. Instead we should return the set